recurly/resource.py

```diff
@@ -715,9 +715,12 @@ def to_element(self, root_name=None):
             except KeyError:
                 continue
             # With one exception, type is an element xml attribute, e.g. <billing info type="credit_card"> or <adjustment type="charge">
-            # For billing_info, type property takes precedence over xml attribute when type = bacs, e.g. <billing info><type>bacs</type></billing_info>. 
-            if attrname in self.xml_attribute_attributes and ((root_name != 'billing_info' and attrname == 'type') or (root_name == 'billing_info' and value != 'bacs')):
-                elem.attrib[attrname] = six.text_type(value)
+            # For billing_info, type property takes precedence over xml attribute when type = bacs or becs, e.g. <billing info><type>bacs</type></billing_info>. 
+            if attrname in self.xml_attribute_attributes and (
+              (root_name != 'billing_info' and attrname == 'type') 
+              or (root_name == 'billing_info' and value not in ('bacs', 'becs'))
```

Comment:
Good catch I forgot about this

Reply:
Do we have a test that covers this? do we need to update it?

Reply:
https://github.com/recurly/recurly-client-python/pull/415/files#diff-c24f9f74a93cc9e07203da5e9e0522c3R684-R716 covers it. It's a repeat of the BACS test since the two are so similar. Is there a better approach?